tests/Behat/Tests/Config/ProfileTest.php

```diff
@@ -188,18 +189,10 @@ public function testSettingShowOutputOption(): void
                 'progress' => [
                     'timer' => true,
                     'show_output' => 'yes',
+                    'short_summary' => false,
                 ],
                 'junit' => false,
             ],
         ], $profile->toArray());
     }
-
```

Comment:
This test is not needed any longer because now we accept the `InSummary` option for the pretty printer